Makefile

```diff
@@ -5,7 +5,7 @@ install-ci-deps:
 	go install github.com/google/go-jsonnet/cmd/jsonnet@v0.20.0
 	go install github.com/google/go-jsonnet/cmd/jsonnetfmt@v0.20.0
 	go install github.com/google/go-jsonnet/cmd/jsonnet-lint@v0.20.0
-	go install github.com/monitoring-mixins/mixtool/cmd/mixtool@a8984473edc33da685bd5f2138eab9cf6a0ff2b7
+	go install github.com/monitoring-mixins/mixtool/cmd/mixtool@7472b3fc310e9d1aaa82d423ae3e7e775a5879f8
```

Comment:
Shouldnt we pin to main?

Reply:
I doubt, as it may break linting without a warning :)

Reply:
I would prefer to have this repo pinned to main actually, as a reminder that things change and always lint against latest (and what we tell people to install). it's a good check to people to realise their local install of mixtool may be out of date

Reply:
ok, let's update for main.